serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -18,52 +20,92 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val genericsNamesWithTypes = Map(
+        ("akka.actor.typed.Behavior", Seq(ClassType.Message)),
+        ("akka.actor.typed.ActorRef", Seq(ClassType.Message)),
+        ("akka.actor.typed.RecipientRef", Seq(ClassType.Message)),
+        ("akka.persistence.typed.scaladsl.ReplyEffect", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.projection.eventsourced.EventEnvelope", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.persistence.typed.scaladsl.Effect", Seq(ClassType.PersistentEvent)))
```

Comment:
Does it make sense to add their `javadsl` counterparts as well?

Reply:
No, because `javadsl` is used in java code, which the plugin will never see.